### ISSUE TITLE
[A11y] Various Pages - Mobile - Filters "hamburger menu"

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1985]
### Operating System

Windows

### Browser

Edge

### Affected page

Homepage

### Admin user?

Yes

### Issue Description

The "hamburger menu" is hard to see when it has focus. I tested on both Edge and Firefox and it has the same colors in both places.

Not focused:

![image](https://user-images.githubusercontent.com/2157974/130695933-6158402f-2266-4e04-bfc0-1747313418bf.png)

Focused:

![image](https://user-images.githubusercontent.com/2157974/130695980-f8a1c979-e936-41f9-a4cb-760637e9eba0.png)

The difference in the two states is minimal and fails contrast requirements for different states. The problem is also present on mouse hover, but that's not a failure since you get a "pointer" mouse cursor.

### Suggested Fix

Consider inverting the colors, as that's likely the easiest way to make this accessible. Otherwise, the styling probably has to get drastic, like having a white border with a black border around it. The background grey makes it difficult to have a focus style that is both a 3:1 contrast ratio against the background *and* 3:1 against the previous state.

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

This is nuanced to test, and would be best done by somebody using something like Accessibility Insights.

**Comment by jechols** [comment]
My wording is somewhat misleading: the focus and hover states are indeed just different enough to be over 3:1, but the focus state blends in to the background at a ratio of *less* than 3:1, and fails for that reason.

**Comment by jechols** [comment]
Actually this may fail in both ways because color alone is used to indicate the state transition. Man this stuff's confusing.

**Comment by azle** [comment]
Updated 12/30/2024 FIXED

Has a focus highlight, blue border around hamburger menu and also works on high contrast view.
<img width="557" alt="Screenshot 2024-12-30 at 7 55 32 AM" src="https://github.com/user-attachments/assets/d529a532-e80b-41f7-b20f-be1f3f99654b" />
